### PR TITLE
Remove not needed parts from Readme

### PR DESCRIPTION
**Pull request by nanbratan**

### CODE REVIEW

**Review by Iogsotot — APPROVED** [review]

File: web/README.md [hunk 1]
```diff
@@ -1,137 +1,50 @@
...
-```
-npm install -g create-react-app
-```
-
-To build and then start the project use:
+## Install
 
 ```
+git clone git@github.com:epam/badgerdoc.git
+cd badgerdoc
```
Comment (Iogsotot) [inline web/README.md:13]: image
I guess you forgot about the `web` subfolder

File: web/README.md [hunk 2]
```diff
@@ -1,137 +1,50 @@
...
+# Learn More
 
-## Test 
-Libraries for testing:
-- React testing library 
-- Jest 
\ No newline at end of file
+This project is using CRA, you can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).\
+To learn about UII, visit [UUI documentation website](https://uui.epam.com/).\
+To learn React, check out the [React documentation](https://reactjs.org/).
```
Comment (Iogsotot) [inline web/README.md:50]: new link for v17 - [legacy.reactjs.org](https://legacy.reactjs.org/docs/hello-world.html)
for v18+ - [react.dev](https://react.dev/)

**Comment by Iogsotot** [comment]
It seems to me that it would be great to add a description of the code style (or a link to it), instructions for writing and running tests, their description

File: web/README.md [hunk 3]
```diff
@@ -1,137 +1,50 @@
...
-For that you need to copy content of `.env.example` file to `.env.local` and specify provided variables.
+# Learn More
 
-## Test 
-Libraries for testing:
-- React testing library 
-- Jest 
\ No newline at end of file
+This project is using CRA, you can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).\
+To learn about UII, visit [UUI documentation website](https://uui.epam.com/).\
```
[the remark this answers is not in the document]
Comment (nanbratan) [reply web/README.md:49]: They are line-breaks

**Review by cakeinsauce — CHANGES_REQUESTED** [review]
Adjust commit message before merging, please
https://github.com/epam/badgerdoc/blob/main/CONTRIBUTING.md

**Review by khyurri — APPROVED** [review]

**Comment by khyurri** [comment]
> Adjust commit message before merging, please https://github.com/epam/badgerdoc/blob/main/CONTRIBUTING.md

Let's automate this checks in https://github.com/epam/badgerdoc/issues/7

**Pull request closed without merging**